sigma/validators/core/tags.py

```diff
@@ -102,87 +102,63 @@ def validate(self, rule: SigmaRule) -> List[SigmaValidationIssue]:
 
 
 @dataclass
-class InvalidCVETagIssue(SigmaValidationIssue):
-    description: ClassVar[str] = "Invalid CVE tagging"
+class InvalidNamespaceTagIssue(SigmaValidationIssue):
+    description: ClassVar[str] = "Invalid tagging namespace"
     severity: ClassVar[SigmaValidationIssueSeverity] = SigmaValidationIssueSeverity.MEDIUM
     tag: SigmaRuleTag
 
 
-class CVETagValidator(SigmaTagValidator):
-    """Validate rule CVE tag"""
+class NamespaceTagValidator(SigmaTagValidator):
+    """Validate rule tag namespace"""
+
+    allowed_namespace = {"attack", "car", "stp", "cve", "tlp", "detection"}
 
     def validate_tag(self, tag: SigmaRuleTag) -> List[SigmaValidationIssue]:
-        tags_pattern = re.compile(r"\d+\-\d+$")
-        if tag.namespace == "cve" and tags_pattern.match(tag.name) is None:
-            return [InvalidCVETagIssue([self.rule], tag)]
+        if tag.namespace not in self.allowed_namespace:
+            return [InvalidNamespaceTagIssue([self.rule], tag)]
         return []
 
 
 @dataclass
-class InvalidDetectionTagIssue(SigmaValidationIssue):
-    description: ClassVar[str] = "Invalid detection tagging"
+class InvalidPatternTagIssue(SigmaValidationIssue):
+    description: ClassVar[str] = "The tag use an invalid pattern"
```

Comment:
"The tag use an invalid pattern" -> "The tag used an invalid pattern" or "The tag is using an invalid pattern" (depending on context, 2nd one is probably more accurate)

Reply:
Sorry for being _that guy_ who corrects grammar in a PR